src/playerhud.py

```diff
@@ -48,6 +53,11 @@ def __init__(
         self.health_sprite.center_x = hud_x
         self.health_sprite.center_y = hud_y
 
+        if isinstance(self.player.game_mode_state, StockGameModePlayerState):
```

Comment:
I think we should add a little explainer on this that the game is drawing at least one heart for empty game mode

Reply:
I changed it so that it no longer draws hearts for empty game mode